lib/features/reps_core/reps_core.views.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Code for the REPS core feature.
+ */
+
+/**
+ * Implements hook_views_query_alter().
+ */
+ function reps_core_views_query_alter(&$view, &$query) {
+
+  if ($view->name == 'reps_bean_blocks_global' && $view->current_display == 'block_reps_bean') {
+    foreach ($query->where as $index => &$condition_group) {
+      foreach ($condition_group['conditions'] as $id => $condition) {
+        if (trim($condition['field']) == "field_data_field_reps_core_node_reference.field_reps_core_node_reference_target_id = :field_data_field_reps_core_node_reference_field_reps_core_node_reference_target_id") {
+          $condition_group['conditions'][$id + 1]['field'] = 'field_data_field_reps_core_node_reference.field_reps_core_node_reference_target_id';
```

Comment:
Basically here, if you have multiple condition_group['conditions'] you will:
first, override the next item ($id +1)
secondly, add several times your new condition.

Take a look to the way I did it.